emily/handler/tests/integration/limits.rs

```diff
@@ -414,3 +436,135 @@ async fn test_complete_rolling_withdrawal_limit_config_works(
     assert_eq!(global_limits.is_ok(), true);
     assert_eq!(global_limits.unwrap(), limits);
 }
+
+#[tokio::test]
+async fn test_available_to_withdraw_fail_no_chainstate_in_db() {
+    let configuration = clean_setup().await;
+
+    // Set limits
+    let limits = Limits {
+        available_to_withdraw: Some(None),
+        peg_cap: Some(None),
+        per_deposit_minimum: Some(None),
+        per_deposit_cap: Some(None),
+        per_withdrawal_cap: Some(None),
+        rolling_withdrawal_blocks: Some(Some(100)),
+        rolling_withdrawal_cap: Some(Some(10000)),
+        account_caps: HashMap::new(),
+    };
+    // Set some chainstates to make set_limits work
+    let chainstates: Vec<Chainstate> = (0..110)
+        .into_iter()
+        .map(|height| new_test_chainstate(height, height, 0))
+        .collect();
+    let _ = batch_set_chainstates(&configuration, chainstates).await;
+
+    let result = apis::limits_api::set_limits(&configuration, limits.clone()).await;
+    assert!(result.is_ok());
```

Comment:
You probably want to `expect` or `unwrap` here, so that you can see the specific error in the panic if it didn't succeed.

Reply:
Oh, which logs are you referring to?